runtime/CSharp/Antlr4BuildTasks/Antlr4ClassGenerationTaskInternal.cs

```diff
@@ -301,7 +301,7 @@ public bool Execute()
                     string framework = TargetFrameworkVersion;
                     if (string.IsNullOrEmpty(framework))
                         framework = "v2.0";
-                    if (framework.StartsWith("v4.5."))
+                    else if (framework.StartsWith("v4.5.") || framework.Equals("v4.6"))
```

Comment:
:question: Would it be better to use `StartsWith` for v4.6 as well?

:thought_balloon: Another option is creating a `Version` and using its comparison operator:

```csharp
Version frameworkVersion = new Version(framework.Substring(1));
if (frameworkVersion >= new Version(4, 5))
    framework = "v4.5";
```